README.md

```diff
@@ -1 +1,25 @@
-# zak_raf_code_challenge
\ No newline at end of file
+# Lap 4 Project
+
+### Installation
+
+- Cd into relevant folder.
+- run `pipenv install`.
+- run `pipenv shell`.
+
+### Usage
+
+- Inside of the pipenv shell, cd into mainapp.
+- run `python manage.py runserver` and open the link in the terminal.
```

Comment:
This command doesn't work if in mainapp but works fine in the urlshortener folder